spec/views/hyrax/citations/work.html.erb_spec.rb

```diff
@@ -1,88 +1,174 @@
-
 RSpec.describe 'hyrax/citations/work.html.erb', type: :view do
-  let(:object_profile) { ["{\"id\":\"999\"}"] }
-  let(:contributor) { ['Gandalf Grey'] }
-  let(:creator)     { ['Bilbo Baggins', 'Baggins, Frodo'] }
-  let(:solr_document) do
-    SolrDocument.new(
-      id: '999',
-      object_profile_ssm: object_profile,
-      has_model_ssim: ['GenericWork'],
-      human_readable_type_tesim: ['Generic Work'],
-      contributor_tesim: contributor,
-      creator_tesim: creator,
-      license_tesim: ['http://creativecommons.org/licenses/by/3.0/us/'],
-      title_tesim: ['the Roared about the Langs'],
-      based_near_label_tesim: ['London'],
-      date_created_tesim: ['1969']
-    )
-  end
-  let(:ability) { nil }
-  let(:presenter) do
-    Hyrax::WorkShowPresenter.new(solr_document, ability)
-  end
+  context "full work metadata" do
+    let(:object_profile) { ["{\"id\":\"999\"}"] }
+    let(:contributor) { ['Gandalf Grey'] }
+    let(:creator)     { ['Bilbo Baggins', 'Baggins, Frodo'] }
+    let(:solr_document) do
+      SolrDocument.new(
+        id: '999',
+        object_profile_ssm: object_profile,
+        has_model_ssim: ['GenericWork'],
+        human_readable_type_tesim: ['Generic Work'],
+        contributor_tesim: contributor,
+        creator_tesim: creator,
+        license_tesim: ['http://creativecommons.org/licenses/by/3.0/us/'],
+        title_tesim: ['the Roared about the Langs'],
+        based_near_label_tesim: ['London'],
+        date_created_tesim: ['1969']
+      )
+    end
+    let(:ability) { nil }
+    let(:presenter) do
+      Hyrax::WorkShowPresenter.new(solr_document, ability)
+    end
 
-  describe 'citations' do
-    let(:page) { Capybara::Node::Simple.new(rendered) }
-    let(:citation) { page.find(citation_selector) }
-    let(:title_selector) { "#{citation_selector} > i.citation-title" }
-    let(:author_selector) { "#{citation_selector} > .citation-author" }
+    describe 'citations' do
+      let(:page) { Capybara::Node::Simple.new(rendered) }
+      let(:citation) { page.find(citation_selector) }
+      let(:title_selector) { "#{citation_selector} > i.citation-title" }
+      let(:author_selector) { "#{citation_selector} > .citation-author" }
 
-    before do
-      assign(:presenter, presenter)
-      render
-    end
+      before do
+        assign(:presenter, presenter)
+        render
+      end
 
-    context 'in APA style' do
-      let(:citation_selector) { 'span.apa-citation' }
-      let(:formatted_title) { 'the Roared about the Langs.' }
-      # entities will be unescaped
-      let(:authors) { 'Baggins, B., & Baggins, F.' }
+      context 'in APA style' do
+        let(:citation_selector) { 'span.apa-citation' }
+        let(:formatted_title) { 'the Roared about the Langs.' }
```

Comment:
You can back out these common `let` statements to a shared context and avoid a bunch of duplication.